integration/setup/setup.go

```diff
@@ -79,7 +79,7 @@ func Resources(c *client.AzureClientSet, g *framework.Guest, h *framework.Host)
 	}
 
 	{
-		err = h.InstallCertResource()
+		err = h.InstallResource("apiextensions-cert-config-e2e", template.CertConfigE2EChartValues, ":stable")
```

Comment:
I think of moving `template.CertConfigE2EChartValues` to `e2etemplates` and removing `InstallCertResource` from the `framework.Host`.